meilisearch-http/src/extractors/authentication/mod.rs

```diff
@@ -204,12 +206,17 @@ pub mod policies {
                 return None;
             }
 
+            let mut validation = TENANT_TOKEN_VALIDATION.clone();
+            validation.insecure_disable_signature_validation();
+            let dummy_key = DecodingKey::from_secret(b"secret");
```

Comment:
uh, you sure about that?

Reply:
Many changed that in the latest commit of this PR [4aef7c5](https://github.com/meilisearch/meilisearch/pull/2237/commits/4aef7c5ac5a8d9856472ed1621ed74685e4c5cf9) as the tests were not passing. You can ask him if you want, maybe I made a mistake when rebasing.

Reply:
DecodingKey doesn't have default implementation, we could change it to:
```suggestion
            let dummy_key = DecodingKey::from_secret(b"dummy");
```
or:
```suggestion
            let dummy_key = DecodingKey::from_secret(b"");
```